.travis.yml

```diff
@@ -1,19 +1,19 @@
-language: node_js
-sudo: false
-node_js:
-  - "6"
-  - "8"
-  - "10"
-  - "11"
-
-script:
-  - npm run compile
-  - npm run test
-
-after_success:
-  - npm run publish-coverage
-
-
-cache:
-  directories:
-    - "node_modules"
+language: node_js
+sudo: false
+node_js:
+  - "8"
+  - "10"
+  - "11"
+
+script:
+  - npm run compile
+  - npm run test
```

Comment:
I would prefer the `test` to contain the `lint` part first

Reply:
Never use lint step like that, but I guess it would work 😄